### PR TITLE
poco: add version 1.13.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **poco/1.13.0**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
I just started an internal build to generate libmysqlclient/8.1.0.

**Comment by uilianries** [comment]
Missing packages are now generated. I restarted this PR build now. The build results should be available via conan-center-bot comment.

**Comment by uilianries** [comment]
@toge I make a small change to disable net test for Poco. We discourage networking in tests in C3I. It may fail as networking is restricted by Jenkins.

File: recipes/poco/all/conanfile.py [hunk 1]
```diff
@@ -293,6 +312,13 @@ def package(self):
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         rmdir(self, os.path.join(self.package_folder, "cmake"))
         rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+        if self.options.get_safe("with_sql_parser", False):
```
Comment (uilianries) [inline recipes/poco/all/conanfile.py:315]:
```suggestion
        # INFO: missing headers https://github.com/pocoproject/poco/issues/4378
        if self.options.get_safe("with_sql_parser", False):
```

File: recipes/poco/all/patches/1.13.0-0002-mysql-include.patch [hunk 2]
```diff
@@ -0,0 +1,143 @@
+diff --git a/Data/MySQL/include/Poco/Data/MySQL/Binder.h b/Data/MySQL/include/Poco/Data/MySQL/Binder.h
```
Comment (uilianries) [inline recipes/poco/all/patches/1.13.0-0002-mysql-include.patch:1]: I'll check what could be done about this patch. Or Poco is including wrongly, or MySQL recipe could use ["include/mysql"] too.

Comment (uilianries) [reply recipes/poco/all/patches/1.13.0-0002-mysql-include.patch:1]: https://github.com/conan-io/conan-center-index/pull/22126

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by RubenRBS** [comment]
regenerating missing binaries now

**Pull request merged**